### PR TITLE
UserDao Add createNewUser

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

File: dataAccessObjects.py [hunk 1]
```diff
@@ -61,6 +61,13 @@ def checkUserExisted(self,userName):
             + 'WHERE user_name="{userName}"'.format(userName=userName)
         return self.queryOneValue(sql)
 
+    def createNewUser(self,userName,userPassword):
+        cursor = self.db.cursor
+        sql = 'INSERT INTO user (user_name,user_password) VALUES (%s,%s)'
+        data = (userName,userPassword)
+        ret = cursor.execute(sql,data)
```
Comment (Billy4195) [inline dataAccessObjects.py]: I guess the above four lines can be replaced by
```
sql = 'INSERT INTO user (user_name,user_password) VALUES ({userName},{userPassword})'.format(userName=userName,userPassword=userPassword)
self.db.cmd(sql)
```

Comment (Billy4195) [inline dataAccessObjects.py]: I guess the above four lines can be replaced by
```
sql = 'INSERT INTO user (user_name,user_password) VALUES ({userName},{userPassword})'.format(userName=userName,userPassword=userPassword)
self.db.cmd(sql)
```

Comment (stvreumi) [inline dataAccessObjects.py]: I have tried, but it can't work....

Comment (Billy4195) [inline dataAccessObjects.py]: What is the error message?

Comment (stvreumi) [inline dataAccessObjects.py]: The test will return error in the function `register_no_right_user()`

Comment (stvreumi) [inline dataAccessObjects.py]: >I also check the documentation. It seems that only this expression is available when using cursor.execute(). I finally pass SQL string and data tuple to this function.
documentation:
>https://dev.mysql.com/doc/connector-python/en/connector-python-api-mysqlcursor-execute.html

This is the comment I write in the another PR.

Comment (Billy4195) [inline dataAccessObjects.py]: Alright, let us modify it in the future.
Now just extract it to be a Dao method

Comment (stvreumi) [inline dataAccessObjects.py]: Should I add a #TODO comment on it?

Comment (Billy4195) [inline dataAccessObjects.py]: Great idea!

Comment (stvreumi) [inline dataAccessObjects.py]: I have tested it, only this method is available.
Below is the error message when I using cmd function.

**Comment by stvreumi** [comment]
OK, I add the comment. Please help me review again. Thanks!

**Pull request merged**